src/ExportButton.tsx

```diff
@@ -237,15 +261,8 @@ const getStoryPayload = async (
   // Story type complains that there is not title in the story but it's okay
   const storyTitle = (story as any)?.title || storyName;
   const storyId = story?.id;
-
-  const [handleStoryRender, getStoryRenderPromise] = getEventHandlerAsPromise();
-
-  const [handleUpdateQueryParams, getUpdateQueryParams] =
-    getEventHandlerAsPromise();
-
-  api.on(STORY_RENDERED, handleStoryRender);
-
-  api.on(UPDATE_QUERY_PARAMS, handleUpdateQueryParams);
+  const [supportedInitialArgs, unsupportedInitialArgs] =
+    getSupportedInitialArgs(story.argTypes, story.initialArgs);
```

Comment:
Don't pass story.argTypes it's better to pass argTypes ( story.argtypes the raw argtypes set by the user) argTypes are story.argTypes after being processed/patched by storybook